### PR TITLE
Alternate scoring with win/lose

### PR DESCRIPTION
**Pull request by seangeggie**
I left in the old !score command to resolve ambiguities since players can appear in multiple matches when player number is odd.

### CODE REVIEW

File: tourney/main.py [hunk 1]
```diff
@@ -233,6 +233,56 @@ def handle_command(cmd):
...
+                scores = Scores.get()
+                scores.add(myTeam, myScore, theirTeam, theirScore)
+                scores.save()
+                response = "Added scores for T{} v T{}!".format(myMatch[0], myMatch[1])
+              elif len(myMatches) > 1:
+                response = "You appear in multiple matches. Please use explicit scoring with !score."
+              else:
+                response = "Found no unscored matches with you as a player."
+          elif len(myTeams) > 1:
+            response = "You appear in multiple teams. Please use explicit scoring with !score."
```
Comment (netromdk) [inline tourney/main.py]: A player can _never_ appear in multiple teams.

File: tourney/main.py [hunk 2]
```diff
@@ -233,6 +233,56 @@ def handle_command(cmd):
...
+                theirTeamIndex = myMatch[(myMatch.index(myTeamIndex)+1)%2]
+                theirTeam = teams[theirTeamIndex]
+
+                unrecorded_matches.remove(myMatch)
+                state.set_unrecorded_matches(unrecorded_matches)
+                state.save()
+                scores = Scores.get()
+                scores.add(myTeam, myScore, theirTeam, theirScore)
+                scores.save()
+                response = "Added scores for T{} v T{}!".format(myMatch[0], myMatch[1])
```
Comment (netromdk) [inline tourney/main.py]: Maybe it should add which team had which score to clarify things?

File: tourney/main.py [hunk 3]
```diff
@@ -233,6 +233,56 @@ def handle_command(cmd):
...
+              elif len(myMatches) > 1:
+                response = "You appear in multiple matches. Please use explicit scoring with !score."
```
Comment (netromdk) [inline tourney/main.py]: Idea: In this case maybe use the chronologically first one? Normally you don't report scores in other orderings?

Comment (netromdk) [reply tourney/main.py]: And if that's the case then `!score` could be removed perhaps.

File: tourney/main.py (hunk 1 above)
Comment (seangeggie) [reply tourney/main.py]: Not _yet_! Well, okay, probably never :P I'll delete that case.

File: tourney/main.py (hunk 3 above)
Comment (seangeggie) [reply tourney/main.py]: Well, we had a case the other day where I had that PTC meeting and we had to postpone a match... Could happen again?

Comment (netromdk) [reply tourney/main.py]: That's a good point! Yeah, then we need the explicit command still.

**Pull request merged**